### PR TITLE
Add license source URL

### PR DESCRIPTION
**Pull request by bpfh**
Since I needed the license source URL as a reference, I added it to the conversion output. If you think this is unnecessary, feel free to reject.

Thanks for a very useful module!

### CODE REVIEW

**Comment by kemitchell** [comment]
FYI https://github.com/shinnn/spdx-license-ids/blob/master/build.js#L13

**Comment by bpfh** [comment]
Ahh, thanks. But I also needed the full license text, which I suppose is not available over there.

**Comment by kemitchell** [comment]
SPDX maintains full license text files. They're on spdx.org, but not yet exported.

File: test.js [hunk 1]
```diff
@@ -4,5 +4,6 @@ var spdxLicenseList = require('./');
 
 test(function (t) {
 	t.assert(spdxLicenseList.MIT.name === 'MIT License');
+  t.assert(spdxLicenseList.MIT.url  === 'http://www.opensource.org/licenses/MIT');
```
Comment (sindresorhus) [inline test.js]: Don't change the indentation.

**Pull request closed without merging**